test.js

```diff
@@ -914,3 +915,61 @@ describe('Elastic()', () => {
     expect(clientInfo.statusCode).toBe(200);
   });
 });
+
+describe('fetch()', () => {
+  test('fetch(url) should return 200 on success with a GET request', async (done) => {
+    const url = 'https://www.google.ca';
+    nock(url).get('/').reply(200);
+
+    const response = await fetch(url);
+    expect(response.ok).toBe(true);
+    expect(response.status).toBe(200);
+    expect(response.url).toEqual(`${url}/`);
+    done();
+  });
+
+  test('fetch(url) should return 404', async (done) => {
+    const url = 'https://dev.api.telescope.cdot.systems';
+    const api = '/v1/status/21';
+    nock(url).get(api).reply(404);
+
+    const response = await fetch(`${url}${api}`);
+
+    expect(response.ok).toBe(false);
+    expect(response.status).toEqual(404);
+    expect(response.statusText).toBe('Not Found');
+    expect(response.url).toEqual(`${url}${api}`);
+    done();
```

Comment:
You don't need `done()` with these tests.  Because you already decorate them as `async`, you are returning a `Promise` and Jest knows to wait on the `Promise` before your test is complete.

Reply:
Ah, wasn't too sure on this one since the request is asynchronous